lib/nexus-snapshot-version.js

```diff
@@ -0,0 +1,12 @@
+function isNexusSnapshotVersion(version) {
+  var pattern = /(\d+\.)*\d\-SNAPSHOT/;
+  if (version) {
+    return version.match(pattern);
+  } else {
+    return false;
+  }
+}
+
+module.exports = {
+  isNexusSnapshotVersion
```

Comment:
Just call this `isSnapshotVersion`, since "nexus" is redundant (it's already in the module name)